### ISSUE TITLE
ACA Regs for Humans (forking CFPB's eregs app for ACA guidance)

### ISSUE THREAD

**Issue by daguar** [codeforamerica/project-ideas#31]
**1-liner:** Fork the CFPB's sweet [eregulations app](http://eregs.github.io/eregulations/) and plug in the various regs for the Affordable Care Act so that folks can easily navigate them
**Project needs:** Could use another developer, ideally one with Python/Django experience; definitely could use coffee
**Status:** Idea
**Description**:

Here's my logic:
- Given the scope and impact of the ACA, the regulatory guidance around it has been very high-visibility.
- The [Consumer Financial Protection Bureau]() built a sweet app for easily navigating, reading, and annotating regs, and it's completely open source: http://eregs.github.io/eregulations/ 
- Combine the high-visibility of the ACA regs with the fact that it's a pretty narrow slice of all the regs out there, it seems like a redeployment of eRegs for the ACA would be a high-visibility, low-time-cost way to show the power of open source while also making it easier for the many, many people tasked with reading ACA guidance (and maybe there's even a HealthCare.gov narrative tie-in to be had here).

In short, let's replace this:
![screen shot 2014-07-03 at 12 55 13 pm](https://cloud.githubusercontent.com/assets/994938/3475068/0bcc6ffc-02ec-11e4-8d53-fa596be39b9a.png)

With this:
![screen shot 2014-07-03 at 12 58 04 pm](https://cloud.githubusercontent.com/assets/994938/3475101/6262eada-02ec-11e4-8fcf-c76ac3e2de6e.png)

cc @ascott1 @ErieMeyer

**Comment by ErieMeyer** [comment]
Love it!

**Comment by ascott1** [comment]
This is really exciting! 

cc: @cmc333333 @khandelwal @theresaanna @vitrix
